docs/theory/theory.rst

```diff
@@ -186,6 +187,14 @@ ramp function, :math:`H` is the wave height, :math:`F_{exc}` is the frequency
 dependent complex wave-excitation amplitude vector, and :math:`\theta` is the 
 wave direction. 
 
+The mean drift force can optionally be included if coefficients are defined in the BEM data. 
```

Comment:
@akeeste Minor comment, but the text should read "mean drift force, and moment, can" correct?